easybuild/tools/options.py

```diff
@@ -176,7 +171,7 @@ def use_color(colorize, stream=sys.stdout):
     elif colorize == COLOR_ALWAYS:
         return True
     else:
-        assert colorize == COLOR_NEVER, "Argument `colorize` must be one of: %s" % ', '.join(COLOR_OPTIONS)
+        assert colorize == COLOR_NEVER, "Argument `colorize` must be one of: %s" % ', '.join(fancylogger.Colorize)
```

Comment:
This should end with `.join([ COLOR_ALWAYS, COLOR_AUTO, COLOR_NEVER ])` if you insist in decoupling this part from `vsc.fancylogger` i.e. not using the same named constants in EB and `vsc.fancylogger`.

Otherwise just drop `COLOR_*` and use `fancylogger.Colorize` everywhere...